### ISSUE TITLE
Connectivity to gene matcher seems limited atm?

### ISSUE THREAD

**Issue by dnil** [Clinical-Genomics/scout#4757]
Thank you for taking the time to contribute an issue! 

**Describe the bug**

Error when checking MatchMaker matches on gene matcher, but looks ok when trying other services like my gene.

![screenshot_2024-08-19_at_14 32 49_720](https://github.com/user-attachments/assets/03cb3eaa-5757-448e-9b5c-6bef87c79512)

See https://scout.scilifelab.se/cust002/F0050705-4.

**Comment by northwestwitch** [comment]
We got that gene matching email, so I guess something has worked yesterday?

**Comment by dnil** [comment]
Yes, as stated there were no matches but also no errors on matching against MyGene2, Matchbox and Boston Childrens. This looks specific to GeneMatcher.

**Comment by dnil** [comment]
No errors today - perhaps it was a transient thing? Let's close for now.
